doc/examples/tissue_classification.py

```diff
@@ -64,21 +65,19 @@
 
 """
 Then, the smoothness factor of the segmentation. Good performance is achieved
-with values between 0 and 0.5.
+with values between 0 and 0.5. Here we are considering it as 0.1
```

Comment:
If the value changes in the code, then adding this makes it more difficult to maintain.

Reply:
But for the given example the value is same and it should be mentioned for better understandability. At the end of day it is an example tutorial and understandability is more important than flexibility for future changes.

Reply:
Plus it is only a comment mentioning the value used in between 0-0.5 so when value changes in the code the comment can also be changed and this does not make it difficult to maintain.

Reply:
I tend to agree with @jhlegarreta . This does not make it "hard to maintain", as you say, but someone modifying this example might very easily forget to change it. I am not 100% sure that we need to specify the value, since it set directly in the line after, which is quite self explanatory.

Reply:
+1 too, should be removed. By experience, it increases easy mistakes during an update. 
`Code should be self-explanatory because documentation can be easily a lie`.

Can you update this? thank you